data/Templates/Stu3R4/CapabilityStatement/_CapabilityStatement.liquid

```diff
@@ -12,6 +12,7 @@
     },
     {% endif -%}
   ],
+  "fhirVersion": "4.0.1",
```

Comment:
For example, it may appear in ImplementationGuide:
![image](https://user-images.githubusercontent.com/66581117/162685059-22a875d4-58ac-423c-b8cb-f31e560c03a2.png)

Reply:
Might be.  In this first round of update, I only modify the value of "fhirVersion" field with 'N' priority, and plan to update others for template refinement. (Since value change in Stu3 to R4 is nice to have.)